utils/math/src/lib.rs

```diff
@@ -0,0 +1,28 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use num_bigint::BigInt;
+
+pub const PRECISION: u64 = 128;
+
+pub fn poly_val(poly: &[BigInt], x: &BigInt) -> BigInt {
```

Comment:
can you add docs on the public functions (unclear the exact reasoning from logic what it does)